docs/tutorials/multimodal/convert_data_to_coco_format.md

```diff
@@ -0,0 +1,169 @@
+# Convert Data to MSCOCO Format
```

Comment:
Please check previous pr for unsolved comments

Reply:
Will go through the previous pr for the comments.

Reply:
Addressed in ce884dba67c46572c8b5c699980d4cfebf55cdd0